.github/workflows/create-release.yml

```diff
@@ -34,8 +34,6 @@ jobs:
       uses: actions/checkout@v3
       with:
         repository: cloudfoundry/cflinuxfs4
-        # TODO - set this to main
```

Comment:
Do you wanna checkout the exact version for consistency? Also, to let workflow build an older version of stack for some reason?

Reply:
nope, the action will check out the main branch by default which is what we want. It can build old stack versions either way because it pulls the image from dockerhub with the right image

Reply:
it was only pinned because the PR to add the dockerfile/makefile target werent merged yet